lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/parameters/RootFolder.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runtime.yarn.client.parameters;
+
+import org.apache.reef.tang.annotations.Name;
+import org.apache.reef.tang.annotations.NamedParameter;
+
+/**
+ * Directory in the local filesystem to hold resources required to run the driver and submit evaluators.
+ */
+@NamedParameter(doc = "Folder to hold resources of the Unmanaged Driver",
+                default_value = RootFolder.DEFAULT_VALUE, short_name = "root_folder")
+public final class RootFolder implements Name<String> {
```

Comment:
@markusweimer currently pretty much every REEF runtime (e.g. Local, Mesos, Standalone, etc.) has its own `RootFolder` parameter. I gave YARN runtime parameter the same name so we can later merge all of them into one parameter in reef-common.